c2corg_ui/static/js/auth/auth.js

```diff
@@ -173,8 +196,30 @@ app.AuthController.prototype.register = function() {
 /**
  * @export
  */
-app.AuthController.prototype.showNewPassForm = function() {
-  alert('TODO');
+app.AuthController.prototype.requestPasswordChange = function() {
+  var alerts = this.alerts_;
+  /**
+   * @typedef {{
+   *   email: string
+   * }}
+   */
+  var data = this.scope_['requestChangePassword'];
+  this.api_.requestPasswordChange(data.email).then(function() {
+    var msg = alerts.gettext(
+        'We sent you an email, please click on the link to reset password.');
```

Comment:
Perhaps you could add this string to https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/i18n.html to make it add to the strings to translate.